stdlib/stdlib-external/ascutils/numberUtils.ts

```diff
@@ -0,0 +1,1077 @@
+import { decRef, malloc } from './grainRuntime'
+import { allocateString, stringSize, wrapString } from './dataStructures'
+import { GRAIN_GENERIC_HEAP_TAG_TYPE } from './tags';
+import { consoleLog } from './console';
+
+/*
+ * This file is a modified version of AssemblyScript's std/assembly/util/number.ts
```

Comment:
Should we add a license header to this file?

Reply:
@ospencer Ostensibly, yes, but it is not immediately clear to me what the license header should contain. I could put a blurb saying that the original work is Apache 2 licensed by the authors of AssemblyScript, but their license file isn't actually filled in with the copyright year and author names, so I can't really put in a legally-well-founded actual license reference....your thoughts?

Reply:
I think just doing what you said is probably good enough. 👍